src/components/Advertisements/Advertisements.tsx

```
@@ -55,54 +54,59 @@ export default function Advertisements(): JSX.Element {
-              ? new URL(edge.node.mediaUrl.toString())
-              : undefined,
-            startDate: edge.node.startDate
-              ? new Date(edge.node.startDate)
-              : undefined,
-            endDate: edge.node.endDate
-              ? new Date(edge.node.endDate)
-              : undefined,
-          }),
+    if (
+      orgAdvertisementListData &&
+      orgAdvertisementListData.organization?.advertisements?.edges
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)